### PR TITLE
Bscharenberg/mt 37

### PR DESCRIPTION
**Pull request by bryn19**

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -22,7 +22,17 @@
         {{/header.dropdown}}
       {{/banner.header}}
 
-      {{component banner.details}}
+      {{#banner.details}}
```
Comment (kedmundson) [inline app/templates/index.hbs]: Could you include a note about why you made this template change? i.e. what problem does it solve

File: app/routes/application.js [hunk 2]
```diff
@@ -66,8 +66,12 @@ export default Route.extend(ApplicationRouteMixin, {
 
     window.WNYC_LEGACY_LOADER = get(this, 'legacyLoader');
 
-    let pollFunction = () => get(this, 'store').findAll('stream');
-    get(this, 'poll').addPoll({interval: 60 * 1000, callback: pollFunction});
+    let pollFunction = () => {
+      get(this, 'store').findAll('stream');
+
```
Comment (kedmundson) [inline app/routes/application.js]: extra newline

File: app/templates/index.hbs (hunk 1 above)
Comment (bryn19) [reply app/templates/index.hbs]: oftentimes on demo the full metadata profile for pieces isn't present, ie the title of the piece shows up but not the composer, as there is a data dump that producers do each day that only makes it into production. So, maybe 1/3 or 1/4 of the time the banner on demo defaults to the show name, making it difficult to test how accurate the metadata alignment is on demo. This solves that issue by displaying the piece title even when the composer name isn't present

**Comment by bryn19** [comment]
I wrote this above as part of a conversation, but in case that gets lost in a rebase here is the explanation for the component change:
oftentimes on demo the full metadata profile for pieces isn't present, ie the title of the piece shows up but not the composer, as there is a data dump that producers do each day that only makes it into production. So, maybe 1/3 or 1/4 of the time the banner on demo defaults to the show name, making it difficult to test how accurate the metadata alignment is on demo. This solves that issue by displaying the piece title even when the composer name isn't present

File: app/templates/index.hbs (hunk 1 above)
Comment (kedmundson) [reply app/templates/index.hbs]: great, thanks

**Pull request merged**